test/scripts/e2e.sh

```diff
@@ -0,0 +1,290 @@
+  version: v1.5.0
+---
+kind: ClusterConfiguration
+apiVersion: konvoy.mesosphere.io/v1beta2
+metadata:
+  name: base-addons-e2e
+  creationTimestamp: "2020-07-29T14:16:29Z"
+spec:
+  kubernetes:
+    version: 1.17.8
+    networking:
+      podSubnet: 192.168.0.0/16
+      serviceSubnet: 10.0.0.0/18
+      iptables:
+        addDefaultRules: false
+    cloudProvider:
+      provider: aws
+    admissionPlugins:
+      enabled:
+        - AlwaysPullImages
+        - NodeRestriction
+  containerNetworking:
+    calico:
+      version: v3.13.4
+      encapsulation: ipip
+      mtu: 1480
+  containerRuntime:
+    containerd:
+      version: 1.3.4
+  osPackages:
+    enableAdditionalRepositories: true
+  nodePools:
+    - name: worker
+  addons:
+    - configRepository: https://github.com/mesosphere/kubernetes-base-addons
+      configVersion: $CONFIG_VERSION
+      addonsList:
+        - name: awsebscsiprovisioner
+          enabled: true
+        - name: awsebsprovisioner
+          enabled: false
+          values: |
+            storageclass:
+              isDefault: false
+        - name: cert-manager
+          enabled: true
+        - name: dashboard
+          enabled: true
+        - name: defaultstorageclass-protection
+          enabled: true
+        - name: dex
+          enabled: true
+        - name: dex-k8s-authenticator
+          enabled: true
+        - name: elasticsearch
+          enabled: true
+        - name: elasticsearch-curator
+          enabled: true
+        - name: elasticsearchexporter
+          enabled: true
+        - name: external-dns
+          enabled: true
+          values: |
+            aws:
+              region:
+            domainFilters: []
+        - name: flagger
+          enabled: false
+        - name: fluentbit
+          enabled: true
+        - name: gatekeeper
+          enabled: true
+        - name: istio # Istio is currently in Preview
+          enabled: false
+        - name: kibana
+          enabled: true
+        - name: konvoyconfig
+          enabled: true
+        - name: kube-oidc-proxy
+          enabled: true
+        - name: localvolumeprovisioner
+          enabled: false
+          values: |
+            storageclasses:
+              - name: localvolumeprovisioner
+                dirName: disks
+                isDefault: false
+                reclaimPolicy: Delete
+                volumeBindingMode: WaitForFirstConsumer
+        - name: nvidia
+          enabled: false
+        - name: opsportal
+          enabled: true
+        - name: prometheus
+          enabled: true
+        - name: prometheusadapter
+          enabled: true
+        - name: reloader
+          enabled: true
+        - name: traefik
+          enabled: true
+        - name: traefik-forward-auth
+          enabled: true
+        - name: velero
+          enabled: true
+    - configRepository: https://github.com/mesosphere/kubeaddons-conductor
+      configVersion: stable-1.17-1.0.0
```

Comment:
this probably should be master 🤔 
```suggestion
      configVersion: master
```

Reply:
From my perspective no. We want to just test the changes in this repo but not master changes on the others, right?

Reply:
no, we want to test the changes on this _PR_ against the next (to be released) version of the whole system. therefore `master` branches is what we want to test. (and PR branch for the repo under test)